src/views/PIGEAN/GeneSet/store.js

```diff
@@ -16,7 +16,9 @@ export default new Vuex.Store({
     },
     state: {
         geneset: keyParams.geneset,
+        sigma: keyParams.sigma || "sigma2",
```

Comment:
while this is fine, you can also just ignore it if not set, meaning leave it as null, and let the backend take care of it. This way, if the default is change in the future (to 4, for example), the backend will be updated, but we don't have to change any frontend code.